org.opentosca.container.util/src/org/opentosca/container/util/converter/AbstractConversionResult.java

```diff
@@ -0,0 +1,13 @@
+package org.opentosca.container.util.converter;
+
+import org.opentosca.container.util.AbstractResult;
+
+/**
+ * Contains the result of any conversion process.
+ * @author Stefan
+ *
+ * @param <T> The type of the conversion result.
+ */
+public interface AbstractConversionResult<T> extends AbstractResult<T> {

```

Comment:
this is not adding any useful information as it is

Reply:
Not yet but maybe in future. I've tried to separate single steps from abstract code to the resulting implementation as good as possible, which makes maintaing the code easier, if there are changes in the future. This is why I've chosen this 'over object oriented' way of building the interfaces.

Reply:
Don't make it too complex just because this could be useful in future. Keep it simple!